packages/nuxt-locokit/src/paths.ts

```diff
@@ -92,57 +90,13 @@ export const ROUTES_PATH = {
    * Workspace paths
    */
   WORKSPACE: {
-    HOME: '/w',
-    CREATE: '/w/create',
-    DETAIL: '/w/:w',
-    /**
-     * Front office
-     */
-    APP: {
-      HOME: '/app',
-      GROUP: '/app/:group',
-      GROUP_PAGES: '/app/:group/p',
-      GROUP_PAGE: '/app/:group/p/:page',
-      GROUP_PAGE_DETAIL: '/app/:group/p/:page/d',
-    },
-    /**
-     * Back office
-     */
-    ADMIN: {
-      HOME: '/admin', // settings + stats
-      // SETTINGS: '/settings',
-      APP: {
-        HOME: '/app',
-        GROUP: '/:group',
-        GROUP_PAGES: '/p',
-        GROUP_PAGE: '/:page',
-        GROUP_PAGE_DETAIL: '/d',
-      },
-      DATA_PROVIDERS: {
-        HOME: '/dps',
-        SETTINGS: '/:dp/settings',
-        SCHEMA: '/:dp/schema',
-        TABLES: '/:dp/t',
-        TABLE: '/:dp/t/:table',
-        TABLE_VIEWS: '/:dp/t/:table/v',
-        TABLE_VIEW: '/:dp/t/:table/v/:view',
-      },
-      AUTOMATIONS: {
-        HOME: '/automations',
-      },
-      MEDIAS: '/media',
-      PERMISSIONS: {
-        HOME: '/permissions',
-        ACL: '/permissions/:acl',
-      },
-      USERS: {
-        HOME: '/users',
-        USER: '/users/:id',
-      },
-      GROUPS: {
-        HOME: '/groups',
-        GROUP: '/groups/:id',
-      },
```

Comment:
Could you comment only these routes ? 
I find them useful for our future implementations ?

Reply:
I can keep it, I do not guarantee the accuracy of the paths, it is very likely that these evolve.

Reply:
Yes, I prefer. I know they won't be accurated in the mid term.

But this structure is here to remember how to structure our routing.